vllm/model_executor/models/llama.py

```diff
@@ -324,6 +327,9 @@ def forward(
             hidden_states = intermediate_tensors["hidden_states"]
             residual = intermediate_tensors["residual"]
 
+        if is_hpu:
+            import habana_frameworks.torch as htorch
+            htorch.core.mark_step()
```

Comment:
Why do we need this?

Reply:
This is an optimization to cut down on graph compile time. It does not change functionality or throughput on compiled graph. I've removed it now.